src/System.Text.Primitives/System/Text/InvariantParser_signed.tt

```diff
@@ -0,0 +1,321 @@
+﻿<#@ template hostspecific="false" language="C#" #>
+<#@ output extension=".cs" #>
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
```

Comment:
We will need to manually generate the .cs files and keep them updated when changes to the .tt file are made. I'm not sure it's easy or possible to make that kind of thing happen automatically with xproj's.  For the .tt files in corefx I have just used VS to regenerate the .cs files (they are generated when the .tt file is saved).